src/Bicep.Core.Samples/Files/baselines_bicepparam/Extensions/parameters.symbols.bicepparam

```diff
@@ -0,0 +1,7 @@
+using 'main.bicep'
+
+extension k8s with {
```

Comment:
Not sure why the symbol isn't showing up.

Reply:
The [test](https://github.com/Azure/bicep/blob/50f46c2e32cb50306657d1386d8bb363b7081a99/src/Bicep.Core.IntegrationTests/Semantics/ParamsSemanticModelTests.cs#L70) filters out symbols other than `ParameterAssignmentSymbol`, but I'm not sure why. We might want to update the code to include all symbols.